test/server/cards/06-CotE/IkomaOrator.spec.js

```diff
@@ -0,0 +1,64 @@
+describe('Ikoma Orator', function() {
+    integration(function() {
+        describe('Ikoma Orator\'s ability', function() {
+            beforeEach(function() {
+                this.setupTest({
+                    phase: 'conflict',
+                    player1: {
+                        inPlay: ['ikoma-orator'],
+                        hand: ['assassination', 'banzai'],
+                        honor: 12
+                    },
+                    player2: {
+                        inPlay: ['bayushi-liar'],
+                        honor: 11
+                    }
+                });
+                this.ikomaOrator = this.player1.findCardByName('ikoma-orator');
+                this.bayushiLiar = this.player2.findCardByName('bayushi-liar');
+                this.shamefulDisplay = this.player2.findCardByName('shameful-display', 'province 1');
+                this.assassination = this.player1.clickCard('assassination');
+                this.banzai = this.player1.clickCard('banzai');
```

Comment:
You would normally use the `findCardByName` function not the `clickCard` function if you are just after a reference to the card.